model/project.go

```diff
@@ -1200,6 +1200,12 @@ func PopulateExpansions(t *task.Task, h *host.Host, oauthToken, appToken string)
 			expansions.Put("github_author", p.GithubPatchData.Author)
 			expansions.Put("github_commit", p.GithubPatchData.HeadHash)
 		}
+		if p.IsGithubMergePatch() {
+			expansions.Put("github_org", p.GithubMergeData.Org)
+			expansions.Put("github_repo", p.GithubMergeData.Repo)
```

Comment:
Can we please document these as well?

Reply:
They are!

Reply:
I only see github_head_branch here. Are the changes maybe not pushed?
![image](https://github.com/evergreen-ci/evergreen/assets/13104717/42e9cd48-1e7e-4f91-834f-4258f8344301)

Reply:
They already exist for PRs, so were already documented https://github.com/evergreen-ci/evergreen/blob/0ec8aae8aed3bf27b9810b79e4f289a53b2e2682/docs/Project-Configuration/Project-Configuration-Files.md?plain=1#L737-L740

Reply:
🙈